### PR TITLE
Convert to PSR-4

### PR DESCRIPTION
**Pull request by georgeboot**
Currently, you can't use this package with composer as the classes are not actually mapped as per [PSR-4 autoloading standard](https://www.php-fig.org/psr/psr-4/).

In this pull request, I update the code to follow the standard.

### CODE REVIEW

**Comment by georgeboot** [comment]
@apexdodge any update?

**Comment by rozenlicht** [comment]
Looking forward to seeing this merged :+1: Currently using the fork from @georgeboot

File: README.md [hunk 1]
```diff
@@ -11,21 +11,12 @@ Api2Pdf.com is a powerful REST API for instantly generating PDF and Office docum
 
 ## <a name="installation"></a>Installation
 
-Add this repository to your Composer file (you can also find us on [Packagist](https://packagist.org/packages/api2pdf/api2pdf.php)):
```
[the remark this answers is not in the document]
Comment (rozenlicht) [reply README.md:14]: You could add an example paragraph containing the `use` statement to be completely servicing.

**Comment by jacovdbergh** [comment]
Any update on this? Would also like this PR to get merged if possible.

**Pull request merged**